src/components/DataTable/DropDownNodeMenu/DropDownNodeMenu.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
+import ExpandableCard from '../ExpandableCard/ExpandableCard';
+
+/* Provided by Material UI for the width of the drop down menu.
+TO DO: Place this formatting into separate .css file */
+const styles = {
+  customWidth: {
+    width: 600,
+  },
+};
+
+/* Manual input at this stage representing dummy data to be parsed through for each card,
+that represents each node. The intention is to obtain information in the format below and
+to be transferred between the components as props. */
+const nodeMap = [
+  {
+    title: 'Node 263',
+    minPower: 'Min Power: 288 Watts',
```

Comment:
Although just dummy data, could change the map to only store integer values and have the components themselves fill in the text, e.g. 'minPower: 288', and have the component paint the 'Min Power: ' text.
Would reduce the amount of duplication - just something to keep in mind when bringing in real data!